src/ResourceManager/Compute/Commands.Compute/Generated/VirtualMachineScaleSetVM/VirtualMachineScaleSetVMListMethod.cs

```diff
@@ -75,16 +133,22 @@ public partial class NewAzureComputeArgumentListCmdlet : ComputeAutomationBaseCm
     {
         protected PSArgument[] CreateVirtualMachineScaleSetVMListParameters()
         {
-            VirtualMachineScaleSetVMListParameters parameters = new VirtualMachineScaleSetVMListParameters();
+            var pExpandExpression = string.Empty;
```

Comment:
Because the existing generator does not work with null value, and also the generator sets string.Empty for initial values of all other string parameters.